tests/src/test/java/ca/bc/gov/open/jag/efiling/services/SubmissionService.java

```diff
@@ -0,0 +1,74 @@
+package ca.bc.gov.open.jag.efiling.services;
+
+import ca.bc.gov.open.jag.efiling.error.EfilingTestException;
+import ca.bc.gov.open.jag.efiling.helpers.SubmissionHelper;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.specification.MultiPartSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.util.UUID;
+
+public class SubmissionService {

```

Comment:
Good job on the `SubmissionService` ! it represents well the intent of the class